### DISCUSSION TITLE
Use NuxtImg in histoire

### DISCUSSION THREAD

**Discussion by oikalyptus** [histoire-dev/histoire#740] [Q&A]
I’ve been trying all day to get NuxtImg to work with the nuxt plugin without success. Its generating the correct ipx routes but the images are not generated at all.  
Has anyone of you ever make it work?

**Comment by yuuzora** [comment]
Having the same error here, IPX doesn't seems to be working with Histoire. 
I'm not sure how we could go around that at the moment
